tests/index.spec.js

```diff
@@ -1,3 +1,7 @@
+import EventsPool from '../src/index';
```

Comment:
Please provide a build version of `EventsPool` for testing:
```js
import EventsPool from '../lib';
```
The build version is something which is going to be used when you install the library. So it is the target for testing.